faucet.config.js

```diff
@@ -3,11 +3,15 @@ let path = require("path");
 
 module.exports = {
 	styleguide: [{
-		source: "./pages.json",
+		source: "./pages.js",
 		target: "./dist",
 		port: 8080,
 		title: "Example Styleguide",
-		description: "This is an example styleguide"
+		description: "This is an example styleguide",
+		renderers: {
+			html: require("./lib/html"),
+			jsx: require("./lib/complate")
```

Comment:
might as well allow for strings here, `require`ing them automatically? seems a little less noisy/technical

Reply:
Not sure I like this 🤔 Need to think about it. I'm surprised you suggest magic like that 🤔

Reply:
Same reason I argued for https://github.com/moonglum/styleguide/issues/8: [!NMP](https://pinboard.in/u:FND/b:8747269a83b1)

Reply:
Just a reminder that complate/mustache/pug/... will be plugins that are installed as separate packages. Which is why I want to require them here. Also note that `$NAME-complate` is the name of the package, while we want to enter `jsx` as the language for the snippet. I'm not sure how you want to solve this?

Reply:
I think it's totally valid to raise concerns that go beyond a PR. I don't like the "don't think upfront" mantra. Thank you for all your input, it is highly appreciated.

Reply:
🙇